spec/views/catalog/show.html.erb_spec.rb

```diff
@@ -4,9 +4,13 @@
   let(:document) { SolrDocument.new :id => 'xyz', :format => 'a' }
   let(:title) { 'Long title that should be truncated at 50 characters' }
   let(:blacklight_config) { Blacklight::Configuration.new }
+  let(:query_params) { { controller: 'catalog', action: 'show' } }
+  let(:search_state) { Blacklight::SearchState.new(query_params, blacklight_config) }
   before(:each) do
     assign :document, document
     allow(view).to receive(:blacklight_config).and_return(blacklight_config)
+    allow(view).to receive(:search_state).and_return search_state
```

Comment:
The search state is provided by a blacklight controller (https://github.com/projectblacklight/blacklight/blob/4e5af828ded57bc426b640bac35a7f94556eee30/app/controllers/concerns/blacklight/controller.rb#L27)  If you're in a view test, it's using a TestController, which doesn't provide that method. So you've done the right thing here by stubbing it out. 👍